### PR TITLE
Run fyle and c1 theme simultaneouly in local POC

### PR DESCRIPTION
**Pull request by DhaaraniCIT**
## Summary by CodeRabbit

- **New Features**
  - Introduced environment and branding configurations for Fyle and Capital One.
  - Added new scripts to start the app with Fyle and Capital One themes.

- **Documentation**
  - Updated README with instructions for setting up and running the app using the new configurations.

### CODE REVIEW

File: src/environments/c1-environment.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (ashwin1111) [inline src/environments/c1-environment.json:1]: remove committed files from PR

File: angular.json [hunk 2]
```diff
@@ -69,6 +93,14 @@
             },
             "development": {
               "browserTarget": "fyle-integrations-app:build:development"
+            },
+            "port1": {
```
Comment (ashwin1111) [inline angular.json:97]: rename better, also update readme

**Comment by ashwin1111** [comment]
please also put out PR for integrations central

**Comment by DhaaraniCIT** [comment]
> please also put out PR for integrations central

https://github.com/fylein/fyle-integrations-central/pull/74

File: package.json [hunk 3]
```diff
@@ -14,7 +14,9 @@
     "generate:svg-sprite": "svg2sprite ./src/assets/icons ./src/assets/sprites/sprite.svg --stripAttrs fill --stripAttrs stroke --stripAttrs id",
     "stop": "kill $(lsof -t -i:4200) || true",
     "storybook": "ng run fyle-integrations-app:storybook",
-    "build-storybook": "npm run setup:env && npm run setup:branding && ng run fyle-integrations-app:build-storybook"
+    "build-storybook": "npm run setup:env && npm run setup:branding && ng run fyle-integrations-app:build-storybook",
+    "start:fyleport": "ng serve --configuration=fyleport",
```
Comment (ashwin1111) [inline package.json:18]: `start:fyle`

File: package.json [hunk 4]
```diff
@@ -14,7 +14,9 @@
...
+    "start:c1port": "ng serve --configuration=c1port"
```
Comment (ashwin1111) [inline package.json:19]: `start:c1`

File: angular.json [hunk 5]
```diff
@@ -69,6 +93,14 @@
             },
             "development": {
               "browserTarget": "fyle-integrations-app:build:development"
+            },
+            "fyleport": {
```
Comment (ashwin1111) [inline angular.json:97]: "fyle"

File: angular.json [hunk 6]
```diff
@@ -69,6 +93,14 @@
...
+              "browserTarget": "fyle-integrations-app:build:fyleport",
+              "port": 4201
+            },
+            "c1port": {
```
Comment (ashwin1111) [inline angular.json:101]: "c1"

File: angular.json [hunk 7]
```diff
@@ -57,6 +57,30 @@
...
+                  "replace": "src/app/branding/config.json",
+                  "with": "src/app/branding/fyle-config.json"
+                },
+                {
+                  "replace": "src/environments/environment.json",
+                  "with": "src/environments/fyle-environment.json"
+                }
+              ]
+            },
+            "c1port": {
```
Comment (ashwin1111) [inline angular.json:73]: "c1"

File: angular.json [hunk 8]
```diff
@@ -57,6 +57,30 @@
               "extractLicenses": false,
               "sourceMap": true,
               "namedChunks": true
+            },
+            "fyleport": {
```
Comment (ashwin1111) [inline angular.json:61]: "fyle"

File: README.md [hunk 9]
```diff
@@ -29,12 +29,50 @@ Follow instructions mentioned in [Integrations Central](https://github.com/fylei
...
 * Run app
 
     ```bash
     npm start
     ```
 
+* Run app in Fyle theme
+
+    ```bash
+    npm run start:fyleport
```
Comment (ashwin1111) [inline README.md:55]: added comments below

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**